Sprint-1/implement/max.js

```diff
@@ -1,4 +1,28 @@
+
+// check max in elements E.g. max([30, 50, 10, 40]), target output: 50
+// E.g. max(['hey', 10, 'hi', 60, 10]), target output: 60 (sum ignores any non-numerical elements)
+
+// we can use following methods to find max in a element = map, max and reduce method.
+
 function findMax(elements) {
+   const numbers = elements.filter(elements => typeof elements === "number");
+   console.log("only numbers", numbers)
+   
+   if(numbers.length === 0) {
+    return -Infinity
+   }
+   const max = Math.max(...numbers)
+   console.log("max", max)
+   return max
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```